### DISCUSSION TITLE
Open Valid URLs in new tab.

### DISCUSSION THREAD

**Discussion by jugrajsingh** [directus/directus#9277] [Archive]
Is there a option to convert a text field that contains URL to a clickable link that opens in new tab.

Like any google sheets or the excel do with a link or a image.

**Comment by benhaynes** [comment]
Possible duplicate of: https://github.com/directus/directus/discussions/8540#discussioncomment-1425411

We should combine all of these into one `Updated Formatted Display` Discussion.
